apps/vr-tests-react-components/package.json

```diff
@@ -56,8 +56,9 @@
     "@fluentui/react-storybook-addon": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28484&commitId=cf5feb3cc7eeb9b9e8b2439022a3b4be78776a9c&env=prod&clientType=FLUENTUI)


  <details><summary><b>InteractionTag Converged</b> 111 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Brand - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Brand - Dark Mode.hover.chromium.png|0| Added|
|InteractionTag Converged.Brand - Dark Mode.pressed.chromium.png|0| Added|
|InteractionTag Converged.Brand - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Brand - High Contrast.hover.chromium.png|0| Added|
|InteractionTag Converged.Brand - High Contrast.pressed.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled - Dark Mode.hover.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled - Dark Mode.pressed.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled - High Contrast.hover.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled - High Contrast.pressed.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled.default.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled.hover.chromium.png|0| Added|
|InteractionTag Converged.Brand Disabled.pressed.chromium.png|0| Added|
|InteractionTag Converged.Brand.default.chromium.png|0| Added|
|InteractionTag Converged.Brand.hover.chromium.png|0| Added|
|InteractionTag Converged.Brand.pressed.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible.default.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible.hover.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible.pressed.chromium.png|0| Added|
|InteractionTag Converged.Circular With Media.default.chromium.png|0| Added|
|InteractionTag Converged.Circular With Media.hover.chromium.png|0| Added|
|InteractionTag Converged.Circular With Media.pressed.chromium.png|0| Added|
|InteractionTag Converged.Circular.default.chromium.png|0| Added|
|InteractionTag Converged.Circular.hover.chromium.png|0| Added|
|InteractionTag Converged.Circular.pressed.chromium.png|0| Added|
|InteractionTag Converged.Default - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Default - Dark Mode.hover.chromium.png|0| Added|
|InteractionTag Converged.Default - Dark Mode.pressed.chromium.png|0| Added|
|InteractionTag Converged.Default - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Default - High Contrast.hover.chromium.png|0| Added|
|InteractionTag Converged.Default - High Contrast.pressed.chromium.png|0| Added|
|InteractionTag Converged.Default - RTL.default.chromium.png|0| Added|
|InteractionTag Converged.Default - RTL.hover.chromium.png|0| Added|
|InteractionTag Converged.Default - RTL.pressed.chromium.png|0| Added|
|InteractionTag Converged.Default.default.chromium.png|0| Added|
|InteractionTag Converged.Default.hover.chromium.png|0| Added|
|InteractionTag Converged.Default.pressed.chromium.png|0| Added|
|InteractionTag Converged.Disabled - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Disabled - Dark Mode.hover.chromium.png|0| Added|
|InteractionTag Converged.Disabled - Dark Mode.pressed.chromium.png|0| Added|
|InteractionTag Converged.Disabled - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Disabled - High Contrast.hover.chromium.png|0| Added|
|InteractionTag Converged.Disabled - High Contrast.pressed.chromium.png|0| Added|
|InteractionTag Converged.Disabled.default.chromium.png|0| Added|
|InteractionTag Converged.Disabled.hover.chromium.png|0| Added|
|InteractionTag Converged.Disabled.pressed.chromium.png|0| Added|
|InteractionTag Converged.Dismissible - RTL.default.chromium.png|0| Added|
|InteractionTag Converged.Dismissible - RTL.hover.chromium.png|0| Added|
|InteractionTag Converged.Dismissible - RTL.pressed.chromium.png|0| Added|
|InteractionTag Converged.Dismissible.default.chromium.png|0| Added|
|InteractionTag Converged.Dismissible.hover.chromium.png|0| Added|
|InteractionTag Converged.Dismissible.pressed.chromium.png|0| Added|
|InteractionTag Converged.Icon - RTL.default.chromium.png|0| Added|
|InteractionTag Converged.Icon - RTL.hover.chromium.png|0| Added|
|InteractionTag Converged.Icon - RTL.pressed.chromium.png|0| Added|
|InteractionTag Converged.Icon.default.chromium.png|0| Added|
|InteractionTag Converged.Icon.hover.chromium.png|0| Added|
|InteractionTag Converged.Icon.pressed.chromium.png|0| Added|
|InteractionTag Converged.Media - RTL.default.chromium.png|0| Added|
|InteractionTag Converged.Media - RTL.hover.chromium.png|0| Added|
|InteractionTag Converged.Media - RTL.pressed.chromium.png|0| Added|
|InteractionTag Converged.Media.default.chromium.png|0| Added|
|InteractionTag Converged.Media.hover.chromium.png|0| Added|
|InteractionTag Converged.Media.pressed.chromium.png|0| Added|
|InteractionTag Converged.Outline - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Outline - Dark Mode.hover.chromium.png|0| Added|
|InteractionTag Converged.Outline - Dark Mode.pressed.chromium.png|0| Added|
|InteractionTag Converged.Outline - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Outline - High Contrast.hover.chromium.png|0| Added|
|InteractionTag Converged.Outline - High Contrast.pressed.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled - Dark Mode.hover.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled - Dark Mode.pressed.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled - High Contrast.hover.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled - High Contrast.pressed.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled.default.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled.hover.chromium.png|0| Added|
|InteractionTag Converged.Outline Disabled.pressed.chromium.png|0| Added|
|InteractionTag Converged.Outline.default.chromium.png|0| Added|
|InteractionTag Converged.Outline.hover.chromium.png|0| Added|
|InteractionTag Converged.Outline.pressed.chromium.png|0| Added|
|InteractionTag Converged.Secondary Text.default.chromium.png|0| Added|
|InteractionTag Converged.Secondary Text.hover.chromium.png|0| Added|
|InteractionTag Converged.Secondary Text.pressed.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small Dismissible.default.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small Dismissible.hover.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small Dismissible.pressed.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small With Icon.default.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small With Icon.hover.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small With Icon.pressed.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small With Media.default.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small With Media.hover.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small With Media.pressed.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small.default.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small.hover.chromium.png|0| Added|
|InteractionTag Converged.Size Extra Small.pressed.chromium.png|0| Added|
|InteractionTag Converged.Size Small Dismissible.default.chromium.png|0| Added|
|InteractionTag Converged.Size Small Dismissible.hover.chromium.png|0| Added|
|InteractionTag Converged.Size Small Dismissible.pressed.chromium.png|0| Added|
|InteractionTag Converged.Size Small With Icon.default.chromium.png|0| Added|
|InteractionTag Converged.Size Small With Icon.hover.chromium.png|0| Added|
|InteractionTag Converged.Size Small With Icon.pressed.chromium.png|0| Added|
|InteractionTag Converged.Size Small With Media.default.chromium.png|0| Added|
|InteractionTag Converged.Size Small With Media.hover.chromium.png|0| Added|
|InteractionTag Converged.Size Small With Media.pressed.chromium.png|0| Added|
|InteractionTag Converged.Size Small.default.chromium.png|0| Added|
|InteractionTag Converged.Size Small.hover.chromium.png|0| Added|
|InteractionTag Converged.Size Small.pressed.chromium.png|0| Added|

</details>


  <details><summary><b>Tag Converged</b> 111 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tag Converged.Brand - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Brand - Dark Mode.hover.chromium.png|0| Added|
|Tag Converged.Brand - Dark Mode.pressed.chromium.png|0| Added|
|Tag Converged.Brand - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Brand - High Contrast.hover.chromium.png|0| Added|
|Tag Converged.Brand - High Contrast.pressed.chromium.png|0| Added|
|Tag Converged.Brand Disabled - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Brand Disabled - Dark Mode.hover.chromium.png|0| Added|
|Tag Converged.Brand Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Tag Converged.Brand Disabled - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Brand Disabled - High Contrast.hover.chromium.png|0| Added|
|Tag Converged.Brand Disabled - High Contrast.pressed.chromium.png|0| Added|
|Tag Converged.Brand Disabled.default.chromium.png|0| Added|
|Tag Converged.Brand Disabled.hover.chromium.png|0| Added|
|Tag Converged.Brand Disabled.pressed.chromium.png|0| Added|
|Tag Converged.Brand.default.chromium.png|0| Added|
|Tag Converged.Brand.hover.chromium.png|0| Added|
|Tag Converged.Brand.pressed.chromium.png|0| Added|
|Tag Converged.Circular Dismissible.default.chromium.png|0| Added|
|Tag Converged.Circular Dismissible.hover.chromium.png|0| Added|
|Tag Converged.Circular Dismissible.pressed.chromium.png|0| Added|
|Tag Converged.Circular With Media.default.chromium.png|0| Added|
|Tag Converged.Circular With Media.hover.chromium.png|0| Added|
|Tag Converged.Circular With Media.pressed.chromium.png|0| Added|
|Tag Converged.Circular.default.chromium.png|0| Added|
|Tag Converged.Circular.hover.chromium.png|0| Added|
|Tag Converged.Circular.pressed.chromium.png|0| Added|
|Tag Converged.Default - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Default - Dark Mode.hover.chromium.png|0| Added|
|Tag Converged.Default - Dark Mode.pressed.chromium.png|0| Added|
|Tag Converged.Default - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Default - High Contrast.hover.chromium.png|0| Added|
|Tag Converged.Default - High Contrast.pressed.chromium.png|0| Added|
|Tag Converged.Default - RTL.default.chromium.png|0| Added|
|Tag Converged.Default - RTL.hover.chromium.png|0| Added|
|Tag Converged.Default - RTL.pressed.chromium.png|0| Added|
|Tag Converged.Default.default.chromium.png|0| Added|
|Tag Converged.Default.hover.chromium.png|0| Added|
|Tag Converged.Default.pressed.chromium.png|0| Added|
|Tag Converged.Disabled - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Disabled - Dark Mode.hover.chromium.png|0| Added|
|Tag Converged.Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Tag Converged.Disabled - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Disabled - High Contrast.hover.chromium.png|0| Added|
|Tag Converged.Disabled - High Contrast.pressed.chromium.png|0| Added|
|Tag Converged.Disabled.default.chromium.png|0| Added|
|Tag Converged.Disabled.hover.chromium.png|0| Added|
|Tag Converged.Disabled.pressed.chromium.png|0| Added|
|Tag Converged.Dismissible - RTL.default.chromium.png|0| Added|
|Tag Converged.Dismissible - RTL.hover.chromium.png|0| Added|
|Tag Converged.Dismissible - RTL.pressed.chromium.png|0| Added|
|Tag Converged.Dismissible.default.chromium.png|0| Added|
|Tag Converged.Dismissible.hover.chromium.png|0| Added|
|Tag Converged.Dismissible.pressed.chromium.png|0| Added|
|Tag Converged.Icon - RTL.default.chromium.png|0| Added|
|Tag Converged.Icon - RTL.hover.chromium.png|0| Added|
|Tag Converged.Icon - RTL.pressed.chromium.png|0| Added|
|Tag Converged.Icon.default.chromium.png|0| Added|
|Tag Converged.Icon.hover.chromium.png|0| Added|
|Tag Converged.Icon.pressed.chromium.png|0| Added|
|Tag Converged.Media - RTL.default.chromium.png|0| Added|
|Tag Converged.Media - RTL.hover.chromium.png|0| Added|
|Tag Converged.Media - RTL.pressed.chromium.png|0| Added|
|Tag Converged.Media.default.chromium.png|0| Added|
|Tag Converged.Media.hover.chromium.png|0| Added|
|Tag Converged.Media.pressed.chromium.png|0| Added|
|Tag Converged.Outline - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Outline - Dark Mode.hover.chromium.png|0| Added|
|Tag Converged.Outline - Dark Mode.pressed.chromium.png|0| Added|
|Tag Converged.Outline - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Outline - High Contrast.hover.chromium.png|0| Added|
|Tag Converged.Outline - High Contrast.pressed.chromium.png|0| Added|
|Tag Converged.Outline Disabled - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Outline Disabled - Dark Mode.hover.chromium.png|0| Added|
|Tag Converged.Outline Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Tag Converged.Outline Disabled - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Outline Disabled - High Contrast.hover.chromium.png|0| Added|
|Tag Converged.Outline Disabled - High Contrast.pressed.chromium.png|0| Added|
|Tag Converged.Outline Disabled.default.chromium.png|0| Added|
|Tag Converged.Outline Disabled.hover.chromium.png|0| Added|
|Tag Converged.Outline Disabled.pressed.chromium.png|0| Added|
|Tag Converged.Outline.default.chromium.png|0| Added|
|Tag Converged.Outline.hover.chromium.png|0| Added|
|Tag Converged.Outline.pressed.chromium.png|0| Added|
|Tag Converged.Secondary Text.default.chromium.png|0| Added|
|Tag Converged.Secondary Text.hover.chromium.png|0| Added|
|Tag Converged.Secondary Text.pressed.chromium.png|0| Added|
|Tag Converged.Size Extra Small Dismissible.default.chromium.png|0| Added|
|Tag Converged.Size Extra Small Dismissible.hover.chromium.png|0| Added|
|Tag Converged.Size Extra Small Dismissible.pressed.chromium.png|0| Added|
|Tag Converged.Size Extra Small With Icon.default.chromium.png|0| Added|
|Tag Converged.Size Extra Small With Icon.hover.chromium.png|0| Added|
|Tag Converged.Size Extra Small With Icon.pressed.chromium.png|0| Added|
|Tag Converged.Size Extra Small With Media.default.chromium.png|0| Added|
|Tag Converged.Size Extra Small With Media.hover.chromium.png|0| Added|
|Tag Converged.Size Extra Small With Media.pressed.chromium.png|0| Added|
|Tag Converged.Size Extra Small.default.chromium.png|0| Added|
|Tag Converged.Size Extra Small.hover.chromium.png|0| Added|
|Tag Converged.Size Extra Small.pressed.chromium.png|0| Added|
|Tag Converged.Size Small Dismissible.default.chromium.png|0| Added|
|Tag Converged.Size Small Dismissible.hover.chromium.png|0| Added|
|Tag Converged.Size Small Dismissible.pressed.chromium.png|0| Added|
|Tag Converged.Size Small With Icon.default.chromium.png|0| Added|
|Tag Converged.Size Small With Icon.hover.chromium.png|0| Added|
|Tag Converged.Size Small With Icon.pressed.chromium.png|0| Added|
|Tag Converged.Size Small With Media.default.chromium.png|0| Added|
|Tag Converged.Size Small With Media.hover.chromium.png|0| Added|
|Tag Converged.Size Small With Media.pressed.chromium.png|0| Added|
|Tag Converged.Size Small.default.chromium.png|0| Added|
|Tag Converged.Size Small.hover.chromium.png|0| Added|
|Tag Converged.Size Small.pressed.chromium.png|0| Added|

</details>


  <details><summary><b>TagGroup Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagGroup Converged.Default.default.chromium.png|0| Added|
|TagGroup Converged.Dismissible With Interaction Tag.default.chromium.png|0| Added|
|TagGroup Converged.Dismissible.default.chromium.png|0| Added|
|TagGroup Converged.Size Extra Small.default.chromium.png|0| Added|
|TagGroup Converged.Size Small.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
verified all :)